doc/userguide/install.rst

```diff
@@ -361,6 +361,34 @@ Suricata packages.
 
     yay -S suricata
 
+
+Auto-Setup
```

Comment:
I think this should go before the section where distro specific installations start.